src/main/java/com/bookmanagement/BookManagement/controller/AuthorController.java

```diff
@@ -0,0 +1,50 @@
+package com.bookmanagement.BookManagement.controller;
+
+import com.bookmanagement.BookManagement.dto.AuthorDto;
+import com.bookmanagement.BookManagement.dto.BookDto;
+import com.bookmanagement.BookManagement.mapper.AuthorMapper;
+import com.bookmanagement.BookManagement.repository.AuthorRepository;
+import com.bookmanagement.BookManagement.repository.BookRepository;
+import com.bookmanagement.BookManagement.service.AuthorService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/authors")
+public class AuthorController {
+
+    private AuthorService authorService;
+    private AuthorRepository authorRepository;
```

Comment:
As a good practice, we generally don't inject repo into the controller. 
The controller always communicates with the service.

Reply:
Done